cfme/tests/intelligence/test_chargeback_assignments.py

```diff
@@ -1,137 +1,138 @@
 # -*- coding: utf-8 -*-
-import cfme.intelligence.chargeback.assignments as cb
 import pytest
 import random
 
-from cfme.intelligence.chargeback.assignments import AssignmentsView
 from cfme import test_requirements
-from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.appliance.implementations.ui import navigate_to, navigator
 
 pytestmark = [
     pytest.mark.tier(3),
     test_requirements.chargeback
 ]
 
 
+@pytest.fixture
+def assigns_collection(appliance):
+    return appliance.collections.assignments
+
+
 @pytest.mark.meta(blockers=[1273654])
-def test_assign_compute_enterprise(appliance, virtualcenter_provider):
+def test_assign_compute_enterprise(appliance, assigns_collection):
     view = navigate_to(appliance.server, 'Chargeback')
 
-    enterprise = cb.Assign(
-        assign_to="The Enterprise",
-        selections={
+    enterprise = assigns_collection.instantiate(assign_to="The Enterprise", selections={
             'Enterprise': {'Rate': 'Default'}
         })
     enterprise.computeassign()
 
     # Assert that the selection made is listed on the UI
-    assign_view = view.browser.create_view(AssignmentsView)
+    assign_view = view.browser.create_view(navigator.get_class(assigns_collection, "All").VIEW)
     row = assign_view.selections.row(name='Enterprise')
     selected_option = row.rate.widget.selected_option
     assert selected_option == "Default", 'Selection does not match'
 
 
-def test_assign_compute_provider(appliance, virtualcenter_provider):
+def test_assign_compute_provider(appliance, virtualcenter_provider, assigns_collection):
     view = navigate_to(appliance.server, 'Chargeback')
 
-    compute_provider = cb.Assign(
+    compute_provider = assigns_collection.instantiate(
         assign_to='Selected Providers',
         selections={
             virtualcenter_provider.name: {'Rate': 'Default'}
         })
     compute_provider.computeassign()
 
-    assign_view = view.browser.create_view(AssignmentsView)
+    assign_view = view.browser.create_view(navigator.get_class(assigns_collection, "All").VIEW)
```

Comment:
REQUIRED: navigator.get_class(assigns_collection, "All").VIEW will return AssignmentsAllView instead of AssignmentsView. Please fix this in all tests.